### ISSUE TITLE
Fracture of the forearm

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5923]
**Preferred term label:** Fracture of the forearm

**Synonyms** Broken forearm 

**Definition (free text, please give PubMed ID)** Bone fracture in the radius, ulna, or wrist.

**Parent term (use hpo.jax.org/app)** Abnormality of forearm bone HP:0040072

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by nicolevasilevsky** [comment]
HP_0003961
